content/blog/tdd-assessment.md

```diff
@@ -26,6 +26,15 @@ As software developers searching the internet is something of a necessity and vi
 
 With that, let's jump right in! Follow the instructions below, take your time to do it well and send us your kata once you’re happy with what you’ve done.
 
+## Things to keep in mind
+
+1. The solution needs to be hosted on a public GitHub/GitLab repository.
+2. You must solve the kata by following best practices for TDD. See [this](https://youtu.be/qkblc5WRn-U) to get started.
```

Comment:
Instead of "You must..." we should say "Follow this link to understand TDD better" or "Watch this video to understand TDD before starting"